python/lsst/ap/association/packageAlerts.py

```diff
@@ -63,6 +67,18 @@ class PackageAlertsConfig(pexConfig.Config):
         default=os.path.join(os.getcwd(), "alerts"),
     )
 
+    doProduceAlerts = pexConfig.Field(
+        dtype=bool,
+        doc="Turn on alert production to kafka if true and if confluent_kafka is in the environment.",
+        default=False,
+    )
+
+    doPackageAlerts = pexConfig.Field(
```

Comment:
Should this be `doWriteAlerts` instead? "doProduce" and "doPackage" sound very similar.